app/lib/core/models/feed_water_rest_info.dart

```diff
@@ -98,6 +107,17 @@ class FeedWaterRestEvent {
       @required this.lastFwrLocation,
       @required this.fwrProvidedOnboard});
 
+  factory FeedWaterRestEvent.defaultFwrEvent(
```

Comment:
- [ ] There should be no default FWR event for sure. Just use `[]` for anything with a list of FWR events.